README.md

```diff
@@ -171,6 +104,13 @@ spec:
 - [Kyverno CLI](documentation/kyverno-cli.md)
 - [Sample Policies](/samples/README.md)
 
+## License
+
+[Apache License 2.0](https://github.com/nirmata/kyverno/blob/master/LICENSE)
+
+## Community Call
```

Comment:
Can we merge both section community call and getting help. We can call it community

Reply:
I consolidated the community sections but kept separate sub-headers for easier reading